### DISCUSSION TITLE
TAB + W should prefer editor tabs , not term tabs

### DISCUSSION THREAD

**Discussion by michelson** [zed-industries/zed#12748] [Feature Requests]
Thank you for this fantastic editor! I've been using it for a week now, and I'm really impressed.

However, there's one issue that's been bothering me. When I try to close a tab, I expect it to be an editor tab, not a terminal tab. I've accidentally closed the app server running in those terminals multiple times because of this.

Is there a way to prevent this or a setting I might have missed? Any help would be greatly appreciated.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
